src/sentry/logging/handlers.py

```diff
@@ -19,6 +19,18 @@ def emit(self, record):
         }
         if record.exc_info:
             kwargs['exc_info'] = record.exc_info
+
+        if '%' in record.msg and record.args:
```

Comment:
Celery logs with `info("%(something)s: %(happened)s", {"something":"mything", "happened": "did stuff"})` and it gets formatted through vanilla logging formatters. Since structlog doesn't do this, we get a bunch of templated strings. This is pretty much a one-step-less-than-blind assertion that it's happening, and an attempt to populate the message.